.circleci/config.yml

```diff
@@ -2366,26 +1622,12 @@ workflows:
      - smoke-standalone-35-rate-limit:
          context: scalyr-agent
       # NOTE: smoke test jobs below still use old smoke tests framework
-     - smoke-docker-json:
-         context: scalyr-agent
      - smoke-docker-api-raw-logs-disabled:
          context: scalyr-agent
```

Comment:
This one should be able to be disabled. Here's a link to a run of the GHA version: https://github.com/scalyr/scalyr-agent-2/runs/7770418121?check_suite_focus=true

Reply:
Thanks - I missed this one + smoke-k8s one. Will remove it from Circle CI config.